### PR TITLE
Acceptance tests functions added to cover new RB admin REST endpoints

### PR DESCRIPTION
**Pull request by CTI777**
Related to https://github.com/Activiti/Activiti/issues/2420

### CODE REVIEW

File: activiti-cloud-acceptance-tests-core/src/main/java/org/activiti/cloud/acc/core/services/runtime/admin/ProcessRuntimeAdminService.java [hunk 1]
```diff
@@ -1,15 +1,20 @@
...
 import org.springframework.hateoas.PagedResources;
 
 public interface ProcessRuntimeAdminService extends BaseService {
 
     @RequestLine("GET /admin/v1/process-instances")
     @Headers("Content-Type: application/json")
     PagedResources<CloudProcessInstance> getProcessInstances();
+    
+    @RequestLine("DELETE /admin/v1/process-instances/{id}")
+    @Headers("Content-Type: application/json")
```
Comment (erdemedeiros) [inline activiti-cloud-acceptance-tests-core/src/main/java/org/activiti/cloud/acc/core/services/runtime/admin/ProcessRuntimeAdminService.java]: I don't think this header is necessary as you are not providing any body content to the request, only the `id` directly in the URL path.

File: activiti-cloud-acceptance-tests-core/src/main/java/org/activiti/cloud/acc/core/services/runtime/admin/TaskRuntimeAdminService.java [hunk 2]
```diff
@@ -1,6 +1,38 @@
...
+    CloudTask updateTask(@Param("taskId") String taskId,
+                         UpdateTaskPayload updateTaskPayload);
+    
+    @RequestLine("POST /admin/v1/{taskId}/assign")
+    CloudTask assign(@Param("taskId") String taskId,
+                     AssignTaskPayload assignTaskPayload);
+    
+    @RequestLine("GET /admin/v1/tasks")
+    @Headers({
+            "Content-Type: application/json",
```
Comment (erdemedeiros) [inline activiti-cloud-acceptance-tests-core/src/main/java/org/activiti/cloud/acc/core/services/runtime/admin/TaskRuntimeAdminService.java]: I don't think `Content-type` is necessary here, `Accept` should be enough.

**Review by erdemedeiros — CHANGES_REQUESTED** [review]
Some small details.

File: activiti-cloud-acceptance-tests-core/src/main/java/org/activiti/cloud/acc/core/services/runtime/admin/TaskRuntimeAdminService.java (hunk 2 above)
Comment (CTI777) [reply activiti-cloud-acceptance-tests-core/src/main/java/org/activiti/cloud/acc/core/services/runtime/admin/TaskRuntimeAdminService.java]: Then it seems that I have to change headers for other RequestLines. I will do refactoring for RB only...

**Review by erdemedeiros — APPROVED** [review]

**Pull request merged**